libraries/AP_RangeFinder/AP_RangeFinder_SRF02I2C.h

```diff
@@ -0,0 +1,64 @@
+#pragma once
+
+#include "RangeFinder.h"
+#include "RangeFinder_Backend.h"
+#include <AP_HAL/I2CDevice.h>
+
+enum {
+    SRF02_I2C_DEFAULT_ADDRESS = 0x70,
+    // http://robot-electronics.co.uk/htm/srf02tech.htm Written measurement value
+    SRF02_I2C_MIN_DISTANCE = 18,  // Not 16cm.
```

Comment:
The spec says:

> The minimum measurement range varies from around 17-18cm (7 inches) on a warm day down to around 15-16cm (6 inches) on a cool day.

So it isn't wrong and there are two registers to read the minimum distance (the autotune registers), have you tested if they work?

In any case, does it ever return a value below 18cm that is wrong? It seems strange to me that it supposedly knows the minimum distance it can read, but then it would give a distance less than that.

Reply:
I did not check the autotune value.
I acquire the value of autotune and make that value the judgment value of minimum distance.